hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheListenerRegistrationMessageTask.java

```diff
@@ -79,7 +79,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "listenerRegistration";
```

Comment:
I think this message task is used by many user methods. However, I think registerListener is a better name because many of the registration methods start with registerListener

Reply:
for the ones which do not directly correspond to one to one user API (this can also be called on the background when client reconnects, etc.) I tried to use the same name as the internal message task name. I tried to do as suggested in [this](https://hazelcast.atlassian.net/browse/SUP-225?focusedCommentId=77717) comment. @kwart Can you also comment what is a proper name for such message tasks based on how the user can use it int he interceptor. I am open to suggestions.